lib/ts/usercontext/index.tsx

```diff
@@ -15,18 +15,31 @@
 
 import React, { useState } from "react";
 import { getNormalisedUserContext } from "../utils";
+import { UserContextType } from "./types";
 
-export const UserContextContext = React.createContext<any>(undefined);
+export const UserContextContext = React.createContext<UserContextType>({
+    userContext: undefined,
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    updateUserContext: () => {},
```

Comment:
We are not going to be providing this function right? Cause in webJS recipe interface, this function is not accessaible. Instead, we said we will be using some implicit observer to detect changes in the object?

Reply:
As discussed we are not adding an observer to respond to changes to userContext anymore